apps/docs/app/features/routes/ressurser/design-tokens/RoundingTokens.tsx

```diff
@@ -20,6 +34,56 @@ export function RoundingTokens(props: BoxProps) {
           rounding på header i appen.
         </Text>
       }
-    ></SharedTokenLayout>
+    >
+      <RoundingTokensTable />
+    </SharedTokenLayout>
   );
 }
+
+type RoundingTokenTableProps = BoxProps;
+
+const RoundingTokensTable = (props: RoundingTokenTableProps) => {
+  const tokenFormatter = useTokenFormatter();
+  return (
+    <Box {...props}>
+      <Table variant="simple" colorScheme="grey">
+        <Thead>
+          <Tr>
+            <Th>Eksempel</Th>
+            <Th>Verdi</Th>
+            <Th>Kode</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {Object.entries(tokens.size["border-radius"]).map(([key, token]) => (
+            <Tr key={key}>
+              <Td>
+                <Box
+                  width={"150px"}
+                  height={"52px"}
+                  borderRadius={key}
+                  border={key}
+                  borderWidth={key}
```

Comment:
Jeg tror muligens ikke `border` og `borderWidth` vil fungere så godt med `key`. Det finnes bare tre verdier for `border`  ("sm", "md" og "lg" - og vel "none" også da), men det finnes en god del flere for `borderRadius`.

Jeg tror du slipper unna med følgende:

```suggestion
                  border="md"
```